packages/public/server/src/module/Renderer/src/Renderer/Renderer.php

```diff
@@ -90,33 +90,37 @@ public function render($input)
         }
 
         $rendered = null;
-        $data = ['state' => $input];
 
-
-        $httpHeader = [
-            'Accept: application/json',
-            'Content-Type: application/json',
-        ];
-
-        $url = $this->getFormatHelper()->isLegacyFormat($input) ? $this->legacyRendererUrl : $this->editorRendererUrl;
-
-        $ch = curl_init();
-        curl_setopt($ch, CURLOPT_URL, $url);
-        curl_setopt($ch, CURLOPT_HTTPHEADER, $httpHeader);
-        curl_setopt($ch, CURLOPT_POST, true);
-        curl_setopt($ch, CURLOPT_POSTFIELDS, json_encode($data));
-        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
-
-        $result = curl_exec($ch);
-        curl_close($ch);
-
-        try {
-            $rendered = json_decode($result, true)['html'];
-        } catch (Exception $e) {
-            $this->sentry->captureException($e, ['tags' => ['renderer' => true]]);
-            throw new RuntimeException(sprintf('Broken pipe'));
+        if ($this->getFormatHelper()->isLegacyFormat($input)) {
```

Comment:
Not sure what's happening here since this file has a syntax error. Reverted the change locally for now